examples/step-90/doc/results.dox

```diff
@@ -0,0 +1,26 @@
+<h1>Results</h1>
+
+The numerical solution for a very finest mesh is shown below.
+
+@image html step-90-solution.png
```

Comment:
This picture is quite large in the PDF you attached. How does it look if embedded in a web page? Perhaps specify a width (by adding ` width=80%` or some such) would be useful.